spring-cloud-gcp-data-datastore/src/main/java/org/springframework/cloud/gcp/data/datastore/repository/query/PartTreeDatastoreQuery.java

```diff
@@ -203,10 +224,26 @@ private Filter getFilter(Object[] parameters) {
 								+ this.queryMethod.getName());
 			}
 		}).toArray(Filter[]::new);
-		return filters.length > 1
+
+		if (builder instanceof ProjectionEntityQuery.Builder) {
```

Comment:
somehow need to insert the equality-compared values into the entities coming back from Datastore for conversion